src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -383,8 +445,34 @@ static void G(A a, B b)
     }
 }";
             var comp2 = CreateCompilation(source2, parseOptions: TestOptions.Regular8, references: new[] { comp.EmitToImageReference() });
```

Comment:
>comp2 [](start = 16, length = 5)

It would be great to get the PE symbols for the implemented interfaces from this compilation and print them out (showing that the `?` annotation was restored from PE).
`Assert.Equal("...", f.Interfaces()[0].ToTestDisplayString());`